css-view-transitions-1/Overview.bs

```diff
@@ -1368,11 +1371,15 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 
 			1. Let |capture| be a new [=captured element=] struct.
 
-			1. Set |capture|'s [=old image=] to the result of [=capturing the image=] of |element|.
+			1. Set |capture|'s [=should generate old pseudo=] to true.
 
 			1. Let |originalRect| be [=snapshot containing block=] if |element| is the [=document element=],
 				otherwise, the element|'s [=border box=].
 
+			1. Let |boundingBox| be |element|'s [=Element/get the bounding box|bounding box=], with [=ink overflow=] taken into account.
```

Comment:
You had it right earlier that if this is the document element then |boundingBox| is the snapshot containing block. Might be simpler to write this as:

if element is document element, capture
else if element's ink overflow rect intersects snapshot containing block, capture.

Reply:
Actually my bad, this makes sense for the root as well. It's ink overflow rect should always intersect with the snapshot containing block if it has a box. And if it doesn't for some obscure case, it makes sense to ignore it.